### PR TITLE
Update 06.know-your-problem.md

### PR DESCRIPTION
**Pull request by cgreene**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

I am not always a big fan of data simulation, but it probably merits a mention. It was in tip 1 (removed in #218) which appears to be the wrong place. This might be a better spot.

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]
LGTM. Only suggestion is adding a reference if you know of any good examples

File: content/06.know-your-problem.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
...
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+One tool in the practicioners toolkit for dealing with datasets that are not thoroughly understood is data simulation.
```
Comment (rasbt) [inline content/06.know-your-problem.md:33]: I think this should be "practitioner's" not "practitioners." Sounds great otherwise.

Comment (Benjamin-Lee) [inline content/06.know-your-problem.md:33]:
```suggestion
One tool in the practicioner's toolkit for dealing with datasets that are not thoroughly understood is data simulation.
```

Comment (Benjamin-Lee) [reply content/06.know-your-problem.md]: Thanks @rasbt! Fixed it.

**Comment by Benjamin-Lee** [comment]
Just circling back on this to get a review. Can one of you take a look at the suggested changes?

File: content/06.know-your-problem.md [hunk 2]
```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
 
 In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
```
Comment (rasbt) [inline content/06.know-your-problem.md:31]: Maybe changing "It is important to note that often molecular or imaging datasets require appropriate clinical"

to 

"It is important to note that molecular or imaging datasets often require appropriate clinical"

will improve readability a bit

File: content/06.know-your-problem.md [hunk 3]
```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
...
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+One tool in the practicioner's toolkit for dealing with datasets that are not thoroughly understood is data simulation.
```
Comment (rasbt) [inline content/06.know-your-problem.md:33]: "practicioner's" should be "practitioner's"

File: content/06.know-your-problem.md [hunk 4]
```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
...
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+One tool in the practicioner's toolkit for dealing with datasets that are not thoroughly understood is data simulation.
+Simulating data under reasonable assumptions and assessing the extent to which the model is capable of capturing or mitigating issues with certain features of the data is a valuable technique for identifying model architectures and hyperparameters that are reasonable under those assumptions.
```
Comment (rasbt) [inline content/06.know-your-problem.md:34]: Just a small mod, but also here, changing to 

"the model is capable of capturing or mitigating issues with certain features"

 to 

 "the model can capture or mitigate issues with certain features"

could make it a tad more readable.

**Review by rasbt — CHANGES_REQUESTED** [review]
Sorry, busy semester. Thanks for making all these changes! Sounds good overall. Below are 2 suggestions and 1 important fix ("practicioner" typo):

**Comment by Benjamin-Lee** [comment]
Thanks @rasbt for the fixes, hope you're doing well. Changes are merged.

**Review by rasbt — APPROVED** [review]
Looks good to me know! And hope you are doing well, too.

**Pull request merged**